openfisca_us/variables/usda/snap.py

```diff
@@ -168,10 +168,3 @@ def formula(spm_unit, period, parameters):
             "snap_expected_contribution_towards_food", period
         )
 
-
-class is_disabled_or_elderly_for_snap(Variable):
```

Comment:
tests currently rely on this (though they're probably broken since it doesn't factor into anything right now). so leave it in for this PR and delete in a subsequent PR that changes the shelter deduction code.